indico/modules/events/management/templates/_base_person.html

```diff
@@ -0,0 +1,51 @@
+<div class="i-form no-max-width event-person-list js-event-person-list">
+    <div class="flashed-messages"></div>
+    <div>
+        <div class="toolbar space-after">
+            <a class="icon-checkbox-checked i-button arrow js-dropdown" href="#" data-toggle="dropdown"></a>
+            <ul class="dropdown">
+                <li>
+                    <a href="#" data-select-all="#person-list-table input.select-row:visible">
+                        {% trans %}All{% endtrans %}
```

Comment:
Sorry, by enclosed I mean that **text** is surrounded with extra whitespace. It doesn't matter if you have:
``` html
<div>
    {% if whatever %}
        {% set ... %}
    {% endif %}
    <span>...</span>
</div>
```
but it does matter if you have
``` html
<div>
    {% trans %}
        Dis is de text
    {% endtrans %}
</div>
```
Check the HTML generated in your browser and you'll see that in the first case not much changes (the browser removes the extra white space between HTML tags), whereas in the second it will keep the whitespace as part of the text.